packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -325,7 +326,7 @@ export interface AutocompleteProps<
    * @returns {ReactNode}
    */
   renderOption?: (
-    props: React.HTMLAttributes<HTMLLIElement>,
+    props: UseAutocompleteGetOptionPropsReturnValue,
```

Comment:
Not sure if it's better to create a type like `AutocompleteRenderOptionProps` or leave as is.   Thoughts?

Reply:
I would leave as is for now.